testsuite/pom.xml

```diff
@@ -8,6 +8,9 @@
         <relativePath>..</relativePath>
     </parent>
     <artifactId>testsuite</artifactId>
+    <properties>
```

Comment:
Put the property into root pom.xml pls, somewhere near https://github.com/quarkus-qe/quarkus-startstop/blob/main/pom.xml#L25